build.sbt

```diff
@@ -135,3 +135,5 @@ riffRaffArtifactResources := Seq(
   baseDirectory.value / "riff-raff.yaml" -> "riff-raff.yaml",
   baseDirectory.value / "cdk/cdk.out/typerighter.template.json" -> "typerighter-cloudformation/typerighter.template.json"
 )
+
+riffRaffManifestProjectName := s"Editorial Tools::Typerighter"
```

Comment:
When using `riffRaffNotifyTeamcity`, the project name defaults to the TeamCity project name (Editorial Tools::Typerigher).

When using `riffRaffUpload`, the project name defaults to the SBT project name (typerighter).

We need to override the default to remain consistent.

Reply:
It's arguably nicer to store this in code, anyhow – it means this project will turn up in a search if people are working backwards from a deploy.